.github/workflows/build-images.yaml

```diff
@@ -45,7 +52,7 @@ jobs:
             # generate v5 tag
             type=semver,pattern={{major}}
           flavor: |
-            latest=auto
+            latest=${{ github.ref == steps.get-latest-tag.outputs.tag }}
```

Comment:
I find it hard to understand, but `latest` should be either `true`, `false`, or `auto` according to https://github.com/docker/metadata-action#latest-tag ?

Reply:
yes and auto is broken, so now with scripting set to false/true